packages/main/src/RangeSlider.js

```diff
@@ -119,6 +126,42 @@ class RangeSlider extends SliderBase {
 		return this.endValue.toFixed(stepPrecision);
 	}
 
+	get _ariaDisabled() {
+		return this.disabled || undefined;
+	}
+
+	get _ariaLabelledByText() {
+		return this.i18nBundle.getText(RANGE_SLIDER_ARIA_DESCRIPTION);
+	}
+
+	get _ariaStartHandleText() {
```

Comment:
the logic in _ariaStartHandleText & _ariaEndHandleText is the same, the only difference is the message bundle var...could we try to extract the common logic somehow?